tests/test_moe.py

```diff
@@ -2,6 +2,7 @@
 import pytest
 import torch
 
+import hivemind
```

Comment:
Better to use a more specific import:
```suggestion
from hivemind.utils import get_dht_time
```

Reply:
done